src/main/java/com/dev/cinema/dao/daoimpl/MovieSessionDaoImpl.java

```diff
@@ -18,7 +18,7 @@
     public List<MovieSession> findAvailableSessions(Long movieId, LocalDate date) {
         try (Session session = HibernateUtil.getSessionFactory().openSession()) {
             Query<MovieSession> getAvailableMovieSessionsQuery = session
-                    .createQuery("FROM MovieSession WHERE movie.id = :movieId "
+                    .createQuery("FROM MovieSession WHERE id = :movieId "
```

Comment:
why have you changed this?

Reply:
I've had problems with sql query in DB, so i searched and refactored code. While were trying to solve one problem, deleted many-to-one movie field here. Haven't noticed it, thanx for paying attention to this.